skmonitor/src/Id.sk

```diff
@@ -0,0 +1,148 @@
+module SKMonitor;
+
+const kINVALID_TRACE_ID: String = "00000000000000000000000000000000";
+const kINVALID_SPAN_ID: String = "0000000000000000";
+
+base class Id private () {
```

Comment:
Is it possible to use any of the KSUID stuff that Josh built? This would give us unique ids that sort by time - pretty useful for logging.

Reply:
That's what I wanted to do.
Unfortunately, KSUID does not follow the TraceId or SpanId specifications.